resources/zib/zib-VisualAcuity.xml

```diff
@@ -0,0 +1,301 @@
+      <path value="Observation.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.extension:visualAcuityMeasurementDevice">
+      <path value="Observation.extension" />
+      <sliceName value="visualAcuityMeasurementDevice" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/observation-deviceCode" />
+      </type>
+    </element>
+    <element id="Observation.extension:visualAcuityMeasurementDevice.value[x]">
+      <path value="Observation.extension.value[x]" />
+      <short value="VisualAcuityMeasurementDevice" />
+      <definition value="The device used measuring the visual acuity." />
+      <alias value="VisusMeetHulpmiddel" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.19.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.6" />
+        <comment value="VisualAcuityMeasurementDevice" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="16830007" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="VisualAcuityDateTime" />
+      <definition value="The date and time when the visual acuity was measured." />
+      <alias value="VisusDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.4" />
+        <comment value="VisualAcuityDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="DecimalVisualAqcuity" />

```

Comment:
'Aqcuity' should be replaced by 'Acuity' (this is an error in the zib itself).

Reply:
Automatically added -> zib-ticket (probably isn't changed until next release). Because zib VisualAcuity is a new zib, there are more issues that are being addressed. I've added these remarks to https://bits.nictiz.nl/browse/ZIB-1445